### PR TITLE
Projection Visualizer | Enhancements

### PR DESCRIPTION
**Pull request by felipemfp**
* Improve tooltips
* Show timeline chart alongside with projection chart
* Deprecate dynamic `shade` for useSeriesColors hook

### CODE REVIEW

**Comment by felipemfp** [comment]
@chicobentojr if you approve, merge both PRs.

File: src/utils/math.ts [hunk 1]
```diff
@@ -0,0 +1,6 @@
+export const median = (raw: number[]) => {
+  const arr = [...raw].sort();
+  const mid = Math.floor(arr.length / 2),
+    nums = [...arr].sort((a, b) => a - b);
```
Comment (chicobentojr) [inline src/utils/math.ts]: Here `arr` can be changed to `raw` and then `arr` variable can be replaced to `nums`, don't?

File: src/components/ProjectionsEditor.tsx [hunk 2]
```diff
@@ -25,6 +26,7 @@ export default function ProjectionsEditor() {
         return (
           <Fragment>
             <ProjectionsChart ref={ref} {...options} height={350} />
+            <CustomizableChart ref={ref} {...options} height={350} chartType="line" />
```
Comment (chicobentojr) [inline src/components/ProjectionsEditor.tsx]: Maybe we can enable `chartType` as an option to the user can choose. What do you think?

**Review by chicobentojr — APPROVED** [review]
@felipemfp LGMT! I did some changes. If you agree with you can merge.

File: src/utils/math.ts (hunk 1 above)
Comment (felipemfp) [reply src/utils/math.ts]: We don't need this line. You're right

Comment (felipemfp) [reply src/utils/math.ts:4]: We don't need this line. You're right

**Comment by felipemfp** [comment]
@chicobentojr I did a couple more changes. It's up to you again

**Comment by chicobentojr** [comment]
Nice changes! I will accept both PRs #21 #20

**Pull request merged**